Update docs

shadichy opened the pull request:
This should remain open until we finish documenting

shadichy committed:
Knowledgebase: userdata-options: More details for Using ramdisk as userdata

Signed-off-by: shadichy <shadichy@blisslabs.org>

shadichy committed:
Knowledgebase: Move pseudo filesystem table from configuration to knowledgebase

Signed-off-by: shadichy <shadichy@blisslabs.org>

shadichy committed:
configuration: initial details for editing kernel command-line parameters

Signed-off-by: shadichy <shadichy@blisslabs.org>

shadichy committed:
docs: reorganize assets and links

- Move top-level images into docs/assets/images/ and group into install/, install/manual/, and install/tools/{rufus,etcher}; strip legacy prefixes.
- Group CSVs under docs/assets/csv/ (cmdline/, fstab/, userdata/) and remove leading prefixes where appropriate.
- Update Markdown references (image links and read_csv() paths) to use the new asset locations.
- Normalize internal links: convert absolute /... links to relative paths and normalize anchor links to the mkdocs format.
- Add .gitignore entries for MkDocs build artifacts: site/, .cache/, .sass-cache/.
- Add small stub pages to restore previously broken internal references: docs/configuration/recovery.md, docs/knowledgebase/grantsu.md.

Signed-off-by: shadichy <shadichy@blisslabs.org>

hmtheboy154 commented:
I don't think we need to put sudo in the editing with BlissOS part ? I don't plan to teach user to download sudo on Termux either.

Also, at the editing parameters using linux part, mention using the bootable installer to edit is an option too.

hmtheboy154 commented:
oh and other unrelated parts related to refactoring are ok

shadichy commented:
> I don't think we need to put sudo in the editing with BlissOS part ? I don't plan to teach user to download sudo on Termux either.

Deck quan tâm, ngta cần 1 editor thì còn j tốt hơn termux tsu + nano đâu

> Also, at the editing parameters using linux part, mention using the bootable installer to edit is an option too.

`from linux`

hmtheboy154 commented:
> Deck quan tâm, ngta cần 1 editor thì còn j tốt hơn termux tsu + nano đâu

well..... mình có nano ở trong OS :))
`vim` nữa. Anh đang tính ở phần nói về root & KSU thì chỉ là vào root bằng `su` rồi dùng `blisspath` để sửa path root cho chuẩn.

> > Also, at the editing parameters using linux part, mention using the bootable installer to edit is an option too.
> 
> `from linux`

cứ làm cái !info để nhắc người ta

shadichy commented:
> > Deck quan tâm, ngta cần 1 editor thì còn j tốt hơn termux tsu + nano đâu
> 
> well..... mình có nano ở trong OS :)) `vim` nữa. Anh đang tính ở phần nói về root & KSU thì chỉ là vào root bằng `su` rồi dùng `blisspath` để sửa path root cho chuẩn.

`/system/bin/su /system/bin/nano <file>`?

> > > Also, at the editing parameters using linux part, mention using the bootable installer to edit is an option too.
> > 
> > 
> > `from linux`
> 
> cứ làm cái !info để nhắc người ta

oge

hmtheboy154 commented:
> > > Deck quan tâm, ngta cần 1 editor thì còn j tốt hơn termux tsu + nano đâu
> > 
> > 
> > well..... mình có nano ở trong OS :)) `vim` nữa. Anh đang tính ở phần nói về root & KSU thì chỉ là vào root bằng `su` rồi dùng `blisspath` để sửa path root cho chuẩn.
> 
> `/system/bin/su /system/bin/nano <file>`?

cũng được, cơ mà anh quen gõ 3 cmd

shadichy committed:
configuration: enhance instructions for editing kernel command-line parameters in BlissOS

- Provided additional information and modified examples

Signed-off-by: shadichy <shadichy@blisslabs.org>

shadichy committed:
configuration: kernel paramerters: use `blisspath` for PATH exports in examples

Signed-off-by: shadichy <shadichy@blisslabs.org>

shadichy committed:
assets: Make `fstab/vars.csv` look more human readable

- Reorganize `fstab/vars.csv` column structure and field description for easier understanding

Signed-off-by: shadichy <shadichy@blisslabs.org>

hmtheboy154 commented on docs/configuration/kernel-parameters.md:
```diff
@@ -5,3 +5,147 @@ order: 4
+
+!!!warning
+
+    This is a little bit more complicated than any of above and can only be applied once you use `grub-android-prober`. If you setup BlissOS for any other bootloaders, or setup BlissOS boot entry yourself, you should know how to do it on your own.
```
@shadichy cho link `grub-android-prober` vào

hmtheboy154 commented on docs/configuration/kernel-parameters.md:
````diff
@@ -5,3 +5,147 @@ order: 4
+su -c 'nano /boot/grub/android.cfg'
+```
+
+![](../assets/images/install/cmdline.png){ width="500" }
````
ê hình như cái ảnh này hơi sai, cái này là placeholder để về sau mình cho ảnh editor vào à

shadichy replied:
对了

shadichy replied:
对了

hmtheboy154 replied:
?

shadichy replied:
là đúng r man

hmtheboy154 replied:
ok.... thế lần này em làm ảnh nhé :v

hmtheboy154 commented on docs/configuration/kernel-parameters.md:
````diff
@@ -5,3 +5,147 @@ order: 4
+sudo umount /data/local/tmp
+```
+
+## Edit command-line parameters from linux
````
from a linux distro

hmtheboy154 commented on docs/configuration/kernel-parameters.md:
```diff
@@ -5,3 +5,147 @@ order: 4
+
+!!!warning
+
+    You're editing from linux, you cannot apply any of the above methods from [Edit command-line parameters from BlissOS](kernel-parameters.md#edit-command-line-parameters-from-blissos).
```
Đẩy cái này lên danger

hmtheboy154 commented on docs/configuration/kernel-parameters.md:
```diff
@@ -5,3 +5,147 @@ order: 4
+
+!!!info
+
+    Since you use linux, we'd expect that you know how to mount partitions and edit files from linux. We'd also expect that you know where you installed BlissOS (your root partition).
```
Đẩy cái này lên warning

```
    Since you are using a linux distro, we'd expect that you know how to mount partitions and edit files using this system. We'd also expect that you know where your BlissOS installation is.
```

hmtheboy154 commented on docs/configuration/kernel-parameters.md:
```diff
@@ -5,3 +5,147 @@ order: 4
+
+!!!info
+
+    This section is only available for uEFI machines.
```
bỏ 2 dòng info này đi cũng được vì anh nghĩ bởi vì người dùng rEFInd là phải có UEFI rồi..... chưa kể anh nghĩ đa phần sẽ phải biết là máy họ đang dùng UEFI

hmtheboy154 commented on docs/configuration/kernel-parameters.md:
```diff
@@ -5,3 +5,147 @@ order: 4
+
+!!!warning
+
+    As mentioned above for BlissOS without bootloader, this applies only if you are using `grub-android-prober`.
```
```
As mentioned in [#without-bootloader](Edit parameters from BlissOS without bootloader), this applies only if you are using [link-cua-project](grub-android-prober).
```

shadichy committed:
docs: update kernel parameters and pseudo filesystem table documentation for clarity and accuracy

- Changes based on #discussion_r2744568704

Signed-off-by: shadichy <shadichy@blisslabs.org>

hmtheboy154 commented on docs/configuration/kernel-parameters.md:
```diff
@@ -90,23 +86,23 @@ sudo nano /data/local/tmp/cmdline.txt
-!!!warning
+!!!danger
 
     You're editing from linux, you cannot apply any of the above methods from [Edit command-line parameters from BlissOS](kernel-parameters.md#edit-command-line-parameters-from-blissos).
```
à quên ko bảo chỗ này

`a linux distro`

shadichy committed:
configuration: Minor details and spell checkings

Signed-off-by: shadichy <shadichy@blisslabs.org>
